osbuild/main_cli.py

```diff
@@ -16,11 +16,7 @@
 import osbuild.monitor
 from osbuild.objectstore import ObjectStore
 
-
-RESET = "\033[0m"
-BOLD = "\033[1m"
-RED = "\033[31m"
-GREEN = "\033[32m"
+from osbuild.markup import RESET, BOLD, RED, GREEN
```

Comment:
Any reason to separate it from the other includes? Shouldn't it just go above `from osbuild.objectstore import ...`? Or maybe even above `osbuild.meta`, depending on how exactly we order it.

Reply:
That fits our current style better, moved it up.